iothub/device/src/Transport/IDeviceIdentity.cs

```diff
@@ -3,15 +3,44 @@
 
 namespace Microsoft.Azure.Devices.Client.Transport
 {
+    /// <summary>
+    /// Interface for device configurations and common attributes.
+    /// </summary>
     internal interface IDeviceIdentity
```

Comment:
since this looks like it is specific to amqp only; if we haven't released yet we could move this to the amqp namespace

Reply:
Sure thing, I will inspect other places it has been used and if they are all in AMQP I will move it

Reply:
Done